src/main/deploy/robot_FailedMerge.xml

```diff
@@ -0,0 +1,1001 @@
+<<<<<<< main

```

Comment:
This looks like a failed merge, not something that should be part of the pull request